### PR TITLE
EX1 oshri ben mosh/natan baraslevski

### PR DESCRIPTION
**Pull request by oshribm88**
ex1

### CODE REVIEW

**Comment by robi-y** [comment]
Shalom Natan & OShri, 
Could not open your .xmi with Eclipse or ArgoUML.
What did you use?
@oshribm88 please add Natan to the conversation

**Comment by oshribm88** [comment]
I can't import the XMI files that i created from ArguUML. therefor i added the ArguUML files instead,include the JPGs.
BTW how can I add Natan to this conversation?

**Comment by robi-y** [comment]
You write his username like this @robi-y 
Comments:
UCD: submit payment: seems like a detail of a bigger use case (clients do not have a goal to submit money!), 
Cloud Storage: is it an external service/ otherwise not part of this diagram

Activity Diagram: the decision node  is used for different scenarios (signup/browse), this is a smell that they should not be together. This diagram is mostly helpful for describing a complex process.

Deployment Diagram: excellent!

Use-case: OK, but sign-up is pretty standard so not much learned about the business from writing it 

OCL: missing

**Comment by oshribm88** [comment]
@natanbr

**Comment by natanbr** [comment]
Thank you Oshri. the @ name thing worked :-)

**Comment by oshribm88** [comment]
Add OCL diagram

**Comment by oshribm88** [comment]
Add EX2

File: hw/Natan&Oshri/EX2/EX2.B.txt [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+----                  --------   ------                -----------           ------------          ------------
+example/data/WordC... BlockBlob  337623                application/octet-... 5/7/2015 4:36:43 P...
+VERBOSE: Transfer Summary
+--------------------------------
+Total:  1.
+Successful: 1.
+Failed: 0.
+
+
+PS C:\> cat ./example/data/WordCountOutput/part-r-00000 | findstr "there"
```
Comment (robi-y) [inline hw/Natan&Oshri/EX2/EX2.B.txt:349]: only needed from this row... but fine - very nice

**Comment by robi-y** [comment]
OCL Fine

**Pull request closed without merging**